app/models/user.js

```diff
@@ -1,8 +1,9 @@
 import Ember from 'ember';
 import DS from 'ember-data';
 
-const { computed, RSVP } = Ember;
+const { computed, PromiseProxyMixin, RSVP } = Ember;
 const { PromiseArray } = DS;
+const ProxyContent = Ember.Object.extend(PromiseProxyMixin);
```

Comment:
Please use DS.PromiseObject for these.  Its the exact same thing, but PromiseObject is what is used everywhere else so it will be easier to maintain over time.